azure/resource-group/variables.tf

```
@@ -11,4 +11,10 @@ variable "tags" {
 variable "environment" {
   description = "nmbrs environment name."
```

Here is a suggestion but completely optional:

```suggestion
  # For a complete list of available Azure regions run at cli:  
  # az account list-locations  --query "[].{displayName:displayName, location:name}" --output table
  description = "(Required) The Azure Region where the Resource Group should exist. Changing this forces a new Resource Group to be created."
````